src/main/resources/templates/ingredients/add.html

```diff
@@ -16,7 +16,7 @@ <h1>Add Ingredients</h1>
     </div>
     <div class="form-group">
         <label>Description
-            <input type="text" description="ingredientDescription" class="form-control">
+            <input type="text" name="ingredientDescription" class="form-control">
```

Comment:
This was causing a Thymeleaf error. Easy mistake to make, but always go through and check all your attributes on your HTML tags to make sure they're the ones you meant to use.